plugin/PDFContextMenu.qml

```diff
@@ -0,0 +1,135 @@
+/****************************************************************************************
+  **
+  ** Copyright (C) 2013-2016 Jolla Ltd., Damien Caliste
+  ** Contact: Martin Jones <martin.jones@jollamobile.com>
+  ** Contact: Damien Caliste <dcaliste@free.fr>
+  ** All rights reserved.
+  ** 
+  ** You may use this file under the terms of BSD license as follows:
```

Comment:
Unfortunately relicensing seems problematic. Would Browser have only Jolla modifications on corresponding component, we could use whatever license, but there seems at least one commit coming outside. Cleanest option would be to rewrite the menu from scratch.